enhancements/cluster-logging/multi-cluster-observability.md

```diff
@@ -0,0 +1,425 @@
+---
+title: multi-cluster-observability-addon
+authors:
+  - "@periklis"
+  - "@JoaoBraveCoding"
+  - "@pavolloffay"
+  - "@iblancasa"
+reviewers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@xperimental"
+  - "@jparker-rh"
+  - "@berenss"
+  - "@bjoydeep"
+approvers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@xperimental"
+api-approvers:
+  - "@alanconway"
+  - "@jcantrill"
+  - "@xperimental"
+creation-date: 2023-11-28
+last-updated: 2023-11-28
+tracking-link:
+  - https://issues.redhat.com/browse/OBSDA-356
+  - https://issues.redhat.com/browse/OBSDA-393
+  - https://issues.redhat.com/browse/LOG-4539
+see-also:
+  - None
+replaces:
+  - None
+superseded-by:
+  - None
+---
+
+# Multi-Cluster Observability Addon
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Multi-Cluster Observability has been an integrated concept in Red Hat Advanced Cluster Management (RHACM) since its inception but only incorporates one of the core signals, namely metrics, to manage fleets of OpenShift Container Platform (OCP) based clusters (See [RHACM Multi-Cluster-Observability-Operator (MCO)](rhacm-multi-cluster-observability)). The underlying architecture of RHACM observability consists of a set of observability components to collect a dedicated set of OCP metrics, visualizing them and alerting on fleet-relevant events. It is an optional but closed circuit system applied to RHACM managed fleets without any points of extensibility.
+
+This enhancement proposal seeks to bring a unified approach to collect and forward logs and traces from a fleet of OCP clusters based on the RHACM addon facility (See Open Cluster Management (OCM) [addon framework](ocm-addon-framework)) by enabling these signals events to land on third-party managed and centralized storage solutions (e.g. AWS Cloudwatch, Google Cloud Logging). The multi-cluster observability addon is an optional RHACM addon. It is a day two companion for MCO and does not necessarily share any resources/configuration with the latter. It provides a unified installation approach of required dependencies (e.g. operator subscriptions) and resources (custom resources, certificates, CA Bundles, configuration) on the managed clusters to collect and forward logs and traces. The addon's name is Multi Cluster Observability Addon (MCOA).
+
```

Comment:
Reflecting a bit on the naming convention. In RHACM today, we have a :

- observability addon and its corresponding operator on the hub called a MCO
- grc addon
- app life cycle addon 

etc

Now calling this Multicluster observability addon would be very confusing. I think I understand the logic behind proposed naming convention -  it is adding on logging and tracing functions to original MCO. But to RHACM customers used to a certain convention, this will be very confusing IMO.

Reply:
Any proposals for a good name? mco-addon?